broadinstitute/pearl
# [JN-451] Add ability to create a new survey

* Adds a new endpoint to configure a survey for a study environment
* Adds new modal for creating a survey in the study admin UI: 
![Screenshot 2023-07-06 at 5 07 03 PM](https://github.com/broadinstitute/juniper/assets/7257391/f1c79bc1-acca-4cb0-a872-05dc6a466cde)

Note that this is the bare minimum needed to unblock survey creation. There are other bits of configuration that we will eventually want to surface to the user: whether or not the survey is required, what the eligibility rules are, survey order, etc.



TO TEST:

1. Boot up Admin API + Admin UI
2. Go to the survey list in the study content manager and add a new survey
3. Verify that a blank survey gets created and it takes you to the survey builder
